src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build and develop your Astro project using any of your favorite code editors, either locally or in the cloud!
+
+Improve your experience by working with `.astro` files in supported editors with Astro language support.
```

Comment:
Too much support?

```suggestion
Improve your experience by working with `.astro` files in editors with Astro language support.
```

Reply:
(Sorry missed the fresh eyes/later memo 😬)

Reply:
Haha, is there EVER too much support??  :)

Fred's pushing some changes too, so we'll wait on that, at least.